packages/xstate-react/src/createHooks.ts

```diff
@@ -0,0 +1,106 @@
+import { createContext, useContext } from 'react';
+import { AnyActorRef, EmittedFrom } from 'xstate';
+import { useActor } from './useActor';
+import { useSelector } from './useSelector';
+
+/**
+ * Creates and returns React hooks that can be used in any component:
+ * - `obj.useActor()` - hook returns the same tuple as `useActor(actorRef)`
+ * - `obj.useSelector(selector, compare?)` - hook returns the same selected value as `useSelector(actorRef, selector, compare)`
```

Comment:
nit: in this file, you are calling the same argument both `compare` and `comparator`